src/services

```diff
@@ -129,7 +129,7 @@ p({
   "serviceScript": """
     return
       new foam.dao.PMDAO(x,
-      new foam.dao.SequenceNumberDAO(1000,
+      new foam.dao.FUIDAO(
```

Comment:
@kgrgreer @jlhughes do we want FUIDAO on localUserDAO from now on?

Reply:
I don't know.

Reply:
Probably not right now. Would like to test and finalize first.